.ci/pipeline_definitions

```diff
@@ -34,6 +34,15 @@ delivery-dashboard:
               image: europe-docker.pkg.dev/gardener-project/releases/ocm-gear/delivery-dashboard
               dockerfile: Dockerfile
               tag_as_latest: True
+              resource_labels:
+              - name: 'gardener.cloud/cve-categorisation'
+                value:
+                  network_exposure: 'public'
+                  authentication_enforced: false
```

Comment:
don't we actually do enforce authentication?

Reply:
as discussed, will keep `authentication_enforced: false`